readme.md

```diff
@@ -102,7 +102,7 @@ within the [doc/koch.rst](doc/koch.rst) file.
 
 ## Contributors
 
-This project exists thanks to all the people who contribute. [Read on to find out how to contribute](#contributing).
```

Comment:
removed "Read on to find out how to contribute](#contributing)" since contributing section is right after